packages/now-cli/src/util/dev/server.ts

```diff
@@ -905,15 +898,22 @@ export default class DevServer {
     ops.push(close(this.server));
 
     if (this.watcher) {
-      this.output.debug(`Closing file watcher`);
+      debug(`Closing file watcher`);
       this.watcher.close();
     }
 
     if (this.updateBuildersPromise) {
-      this.output.debug(`Waiting for builders update to complete`);
+      debug(`Waiting for builders update to complete`);
       ops.push(this.updateBuildersPromise);
     }
 
+    for (const pid of this.devServerPids) {
+      ops.push(this.killBuilderDevServer(pid));
+    }
+
+    // Ensure that the `builders.tar.gz` file has finished extracting
```

Comment:
I thought there was no `builders.tar.gz` anymore?

Reply:
There isn't, I'll update this comment.